ISICbalance/AccountView.swift

```diff
@@ -0,0 +1,109 @@
+        formStackView.addArrangedSubview(usernameLabel)
+        
+        let usernameTextField = UITextField()
+        usernameTextField.backgroundColor = UIColor.MyTheme.textFieldBackground
+        usernameTextField.textColor = UIColor.MyTheme.textColor
+        usernameTextField.tintColor = UIColor.MyTheme.textColor
+        usernameTextField.setLeftPaddingPoints(5)
+        usernameTextField.layer.cornerRadius = 2
+        usernameTextField.layer.masksToBounds = true
+        formStackView.addArrangedSubview(usernameTextField)
+        
+        usernameTextField.snp.makeConstraints { (make) in
+            make.width.equalTo(280)
+            make.height.equalTo(45)
+        }
+        
+        let passwordLabel = UILabel()
+        passwordLabel.text = L10n.Login.password
+        passwordLabel.textColor = UIColor.MyTheme.labelBlue
+        formStackView.addArrangedSubview(passwordLabel)
+        
+        let passwordTextField = UITextField()
+        passwordTextField.backgroundColor = UIColor.MyTheme.textFieldBackground
+        passwordTextField.textColor = UIColor.MyTheme.textColor
+        passwordTextField.tintColor = UIColor.MyTheme.textColor
+        passwordTextField.setLeftPaddingPoints(5)
+        passwordTextField.layer.cornerRadius = 2
+        passwordTextField.layer.masksToBounds = true
+        formStackView.addArrangedSubview(passwordTextField)
+        
+        passwordTextField.snp.makeConstraints { (make) in
+            make.width.equalTo(280)
+            make.height.equalTo(45)
+        }
+        
+        let loginButton = UIButton()
+        loginButton.setTitle(L10n.Login.login, for: .normal)
+        loginButton.setTitleColor(UIColor.MyTheme.backgroundColor, for: .normal)
+        loginButton.backgroundColor = UIColor.MyTheme.labelBlue
+        formStackView.addArrangedSubview(loginButton)
+        
+        loginButton.snp.makeConstraints { (make) in
+            make.width.equalTo(280)
+            make.height.equalTo(45)
+        }
+        
+        let picturesStackView = UIStackView()
+        self.addSubview(picturesStackView)
+        
+        picturesStackView.snp.makeConstraints { (make) in
+            make.leading.equalToSuperview().offset(20)
+            make.trailing.equalToSuperview().offset(-20)
+            make.bottom.equalToSuperview().offset(-30)
+        }
+        
+        let authorPicture = UIImageView(image: UIImage(asset: Asset.janmisar))
+        picturesStackView.addArrangedSubview(authorPicture)
+        
+        let spacer = UIView()
+        picturesStackView.addArrangedSubview(spacer)
+        
+        let companyPicture = UIImageView(image: UIImage(asset: Asset.mightycreations))
+        companyPicture.contentMode = .scaleAspectFit
+        picturesStackView.addArrangedSubview(companyPicture)
+        
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension UITextField {
```

Comment:
Pokud to používáš i jinde, tak si to dej někam do souboru Extensions a nebo to tady dej fileprivate.

Reply:
Bylo přesunuto v branchi MVVM.